Packs/Cymulate/pack_metadata.json

```diff
@@ -14,7 +14,8 @@
     "useCases": [],
     "keywords": [],
     "githubUser": [
-        "maor-b"
+        "maor-b",
+		"GillSamia"
```

Comment:
this list is used to tag the users in changes related to this pack, are you sure you want to add your user here?